src/Components/SmartComponents/SystemsPage/SystemsTableToolbar.js

```diff
@@ -97,7 +98,14 @@ const SystemsTableToolbar = ({
                         parameters.filter,
                         apply
                     ),
-                    excludedFilter(apply, parameters)
+                    excludedFilter(apply, parameters),
+                    useOsVersionFilter(
+                        'rhel_version',
```

Comment:
do we have any case where the url will be dynamic thus you pass it as an argument?